### PR TITLE
[DOCS] Remedy formula display in notebook

### PR DESCRIPTION
**Pull request by ZhiqingXiao**
Remedy the incorrect display of a formula in an example notebook.
(Line changing within the LaTeX codes refrains the formula from being displayed correctly.)

### CODE REVIEW

**Comment by ZhiqingXiao** [comment]
@googlebot I signed it!

**Comment by sjsmarc** [comment]
@googlebot I signed it!

File: tf_quant_finance/examples/jupyter_notebooks/American_Option_Black_Scholes.ipynb [hunk 1]
```diff
@@ -144,8 +144,7 @@
         " ### Setup American Option pricer\n",
         "   American option price $V(x, t)$ of an option with spot price $x$ at time $t$\n",
         "   under Black-Scholes model satisfies the following PDE\n",
-        "   $$V_{t} + \\frac{\\sigma^2}{2}  x^2  V_{xx} + r  x  V_{x}\n",
-        "   - r  V(t, x) = 0.$$\n",
+        "   $$V_{t} + \\frac{\\sigma^2}{2}  x^2  V_{xx} + r  x  V_{x}\n- r  V(t, x) = 0.$$\n",
```
Comment (saxena-ashish-g) [inline tf_quant_finance/examples/jupyter_notebooks/American_Option_Black_Scholes.ipynb]: Thank you for your PR! Can you please remove the newline character which is not needed? Please keep the space before the - though.

Comment (ZhiqingXiao) [reply tf_quant_finance/examples/jupyter_notebooks/American_Option_Black_Scholes.ipynb]: @saxena-ashish-g Thanks for reviewing. I have edited accordingly.

**Pull request merged**